samples/source/500-source.yaml

```diff
@@ -14,6 +14,8 @@ spec:
     secretKeyRef:
       name: rabbitmq-default-user
       key: password
+  connectionSecret:
```

Comment:
Should [this](https://github.com/knative-sandbox/eventing-rabbitmq/tree/main/samples/source#create-the-rabbitmq-source) also be updated to reflect the connectionSecret?

Reply:
You are totally right. I should update the step to add key `uri` to the connection secret as well.